<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-100</x>
      <y>-30</y>
      <width>1024</width>
      <height>701</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>dashboard.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="ACCELERO">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>8</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255,255,255, 90);
text-align: center;
color: red;
font: 8pt &quot;Comic Sans MS&quot;;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>  ACCELEROMETER</string>
    </property>
   </widget>
   <widget class="QLabel" name="DATA">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(100,100,208, 100);\ntext-align: center;\ncolor: red;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="KONDISI">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(100,15,95, 90);
text-align: center;
color: red;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="ImgWidget" native="true">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>400</width>
      <height>180</height>
     </rect>
    </property>
    <widget class="QPushButton" name="manual">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#startButton {
border: none;
border-radius: 20px;
background-color: #fbfbfb;
box-shadow: 0 5px #666;
}
QPushButton:hover#startButton {
border: none;
border-radius: 20px;
background-color: #f1f1f1;
}</string>
     </property>
     <property name="text">
      <string>MANUAL</string>
     </property>
    </widget>
    <widget class="QPushButton" name="startButton_4">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>30</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#startButton {
border: none;
border-radius: 20px;
background-color: #fbfbfb;
box-shadow: 0 5px #666;
}
QPushButton:hover#startButton {
border: none;
border-radius: 20px;
background-color: #f1f1f1;
}</string>
     </property>
     <property name="text">
      <string>AUTO</string>
     </property>
    </widget>
    <widget class="QLabel" name="DATA_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(100,100,208, 100);\ntext-align: center;\ncolor: red;</string>
     </property>
     <property name="text">
      <string>          MODE</string>
     </property>
    </widget>
    <widget class="QLabel" name="tipe">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(100,15,95, 90);
text-align: center;
color: red;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="startButton_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#startButton {
border: none;
border-radius: 20px;
background-color: #fbfbfb;
box-shadow: 0 5px #666;
}
QPushButton:hover#startButton {
border: none;
border-radius: 20px;
background-color: #f1f1f1;
}</string>
    </property>
    <property name="text">
     <string>Start video</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nambah">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>370</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>+ 10</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mundur">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>370</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>- 10</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nambah_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>310</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>+ 25</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mundur_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>310</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>- 25</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stop">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>360</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>STOP</string>
    </property>
   </widget>
   <widget class="QLabel" name="kecepatan">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>30</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(100,15,95, 90);
text-align: center;
color: red;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ACCELERO_2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>8</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255,255,255, 90);
text-align: center;
color: red;
font: 8pt &quot;Comic Sans MS&quot;;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>      KECEPATAN</string>
    </property>
   </widget>
   <widget class="QLabel" name="KONDISI_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>340</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(100,15,95, 90);
text-align: center;
color: red;</string>
    </property>
    <property name="text">
     <string>   KECEPATAN</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
